server/apps/dictionaries/service.py

```diff
@@ -21,34 +21,51 @@
 logger = logging.getLogger(__name__)
 
 
+def words(text):
+    return re.findall('[a-z]+', text.lower())
+
+
+def read(stream):
+    return stream.read().decode('utf-8')
+
+
+def merge(doc, words):
+    doc['content'] = list(set(doc.get('content', []) + words))
+
+
+def read_from_file(doc):
+    content = doc.pop(DICTIONARY_FILE)
+    if 'text/' not in content.mimetype:
+        raise SuperdeskApiError.badRequestError('A text dictionary file is required')
+    return words(read(content))
```

Comment:
Dictionaries have a word on each line and it's safer to read one line, strip it and consider it a word